apm-agent-plugins/apm-jms-plugin/src/test/java/co/elastic/apm/agent/jms/JmsInstrumentationIT.java

```diff
@@ -386,7 +416,10 @@ private void verifyQueueSendReceiveOnNonTracedThread(Queue queue)
         Message incomingMessage = resultQ.poll(2, TimeUnit.SECONDS);
         assertThat(incomingMessage).isNotNull();
         verifyMessage(message, incomingMessage);
-        verifySendReceiveOnNonTracedThread(queue.getQueueName(), outgoingMessage);
+        //noinspection ConstantConditions
+        if (tracer.currentTransaction().isSampled()) {
```

Comment:
what about having the annotation directly on `isSampled` method in order to avoid duplicates of `//noinspection ConstantConditions` ?

Reply:
Not sure what you mean. This removes the warning about `tracer.currentTransaction()` potentially returning `null`